File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float change = get_float("Change owed: "); // asks the user about the change owed //
    while (change < 0)
    {
        change = get_float("Change owed: ");
    }
    int cents = round(change * 100); // multiplies it by 100 so I don't have to use decimals //
    int quarters = cents / 25; //finds quarters
    cents = cents % 25;
    int dimes = cents / 10; //finds dimes
    cents = cents % 10;
    int nickels = cents / 5; //finds nickels
    cents = cents % 5;
    int pennies = cents; //finds pennies
    int coins = quarters + dimes + nickels + pennies;
    printf("%d\n", coins); // prints the amount of coins //
}

File: pset6/cash/cash.py
from cs50 import get_float
change = get_float("Change Owed: ")  # asks how much change is needed

if change < 0:
    bool = True
while (bool == True):
    change = get_float("Change Owed: ")  # keeps on asking until you give a good answer
cents = int(change * 100)  # multiplies it by 100 so I don't have to use decimals
quarters = int(cents / 25)  # sees how many quarters
cents = cents % 25
dimes = int(cents / 10)  # sees how many dimes
cents = cents % 10
nickels = int(cents / 5)  # sees how many nickels
cents = cents % 5
pennies = int(cents)  # sees how many pennies
coins = int(quarters + dimes + nickels + pennies)
print(coins)